tests/integration/cmn/client/acl.go

```diff
@@ -0,0 +1,119 @@
+package client
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"time"
+
+	pb "github.com/anoideaopen/foundation/proto"
+	"github.com/anoideaopen/foundation/test/integration/cmn"
+	"github.com/anoideaopen/foundation/test/integration/cmn/client"
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric/integration/nwo"
+	"github.com/hyperledger/fabric/integration/nwo/commands"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+const MultisignKeyDelimeter = "/"
+
+func CheckUserMultisigned(network *nwo.Network, peer *nwo.Peer, user *UserFoundationMultisigned) {
+	Eventually(func() string {
+		sess, err := network.PeerUserSession(peer, "User1", commands.ChaincodeQuery{
+			ChannelID: cmn.ChannelAcl,
+			Name:      cmn.ChannelAcl,
+			Ctor:      cmn.CtorFromSlice([]string{"checkKeys", user.PublicKey()}),
+		})
+		Eventually(sess, network.EventuallyTimeout).Should(gexec.Exit())
+		Expect(sess.ExitCode()).To(Equal(0))
+		if sess.ExitCode() != 0 {
+			return fmt.Sprintf("exit code is %d: %s, %v", sess.ExitCode(), string(sess.Err.Contents()), err)
+		}
+
+		out := sess.Out.Contents()[:len(sess.Out.Contents())-1] // skip line feed
+		resp := &pb.AclResponse{}
+		err = proto.Unmarshal(out, resp)
+		Expect(err).NotTo(HaveOccurred())
+		if err != nil {
+			return fmt.Sprintf("failed to unmarshal response: %v", err)
+		}
+
+		addr := base58.CheckEncode(resp.GetAddress().GetAddress().GetAddress()[1:], resp.GetAddress().GetAddress().GetAddress()[0])
```

Comment:
I'd introduce a variable addressBytes := resp.GetAddress().GetAddress().GetAddress() to make this line a little shorter